SpecFlow.BDD.Selenium.POM.Automation.Mars/Automation.Mars.POM/Pages/LoginPage.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Automation.Mars.Core.Abstraction;
+using Automation.Mars.Core.Base;
+using Automation.Mars.Core.DIContainer;
+using Automation.Mars.POM.WebAbstraction;
+using BoDi;
+using Serilog;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Automation.Mars.POM.Pages
+{
+    public class LoginPage : TestBase, ILoginPage
+    {
+        IAppConfiguration _iappConfiguration;
+        IDriver _idriver;
+        IAtBy bySignIn => GetBy(LocatorType.XPath, "//a[contains(text(),'Sign In')]");
+        IAtWebElement SignIn => _idriver.FindElement(bySignIn);
```

Comment:
Thanks again.
The reason for implementing my own IWebElement as following.
1. I can implement explicit wait in my own IWebElement class. I don't need to use waits everywhere in page object classes. Only inside my own IWebElement class, it would be reusable.
2. I have implemented logging merchanism inside it. Also can reuse this way to print error stack messages when some exceptions come out and it will not stop the testing. It's good for reviewing running process of test cases.
3. If in a testing team, co-workers will not care about how to handle those exceptions when getting an element and operating those elements. They are all done in the encapsulated framework.